src/components/composer/composer/composer.xml

```diff
@@ -1,6 +1,10 @@
 <templates>
   <t t-name="o-spreadsheet-Composer" owl="1">
-    <div class="o-composer-container">
+    <div
+      class="o-composer-container"
+      t-on-mousedown.stop.prevent=""
```

Comment:
I'm pretty sure most of the `.stop` were from when I used the DOM focus for the style in the composer instead of the focus in the props. But it turns out that they are still useful: 

These one are useful to prevent the loss of focus of the composer when clicking in the autocomplete dropdown. It doesn't happen in practice, since everything in this dropdown have an `onClick` that calls `this.autoComplete`, which triggers a re-render and get the focus back, but if we were to add margins  that do'nt have `onClick` in the dropdown, the `.stop` would be needed, so i'm for keeping them. I moved them in the `o-composer-assistant` div tho, so their role is clearer.